torch/csrc/jit/passes/onnx/onnx_log.h

```diff
@@ -0,0 +1,24 @@
+#pragma once
+#include <torch/csrc/Export.h>
+#include <ostream>
+#include <string>
+
+namespace torch {
+namespace jit {
+
+TORCH_API bool get_onnx_logging_enabled();
+
+TORCH_API void set_onnx_logging_enabled(bool enabled);
+
+TORCH_API void set_onnx_logging_output_stream(std::ostream& out_stream);
+
+TORCH_API std::ostream& get_onnx_logging_output_stream();
```

Comment:
```suggestion
TORCH_API std::ostream& get_onnx_logging_handler();
```

Maybe a rename here to dont leak implementation detail (output stream is an implementation detail)

Reply:
renamed with underscore to mark as internal api. I'll keep it simple for now naming with the return class.